openedx/core/djangoapps/dark_lang/models.py

```diff
@@ -37,9 +38,21 @@ def released_languages_list(self):
         ``released_languages`` as a list of language codes.
 
         Example: ['it', 'de-at', 'es', 'pt-br']
+
+        eduNEXT: we support only the list of available languages from the site
+        otherwise is the same as having no configuration
         """
-        if not self.released_languages.strip():
-            return []
+        if settings.FEATURES.get("EDNX_SITE_AWARE_LOCALE", False):
+            site_released_langs = getattr(settings, "released_languages", None)
+
+            # If there aren't any released_languages configured
+            if not site_released_langs:
+                site_released_langs = []
+            else:
+                site_released_langs = [lang.lower().strip() for lang in site_released_langs.split(',')]
+                site_released_langs.sort()
+
+            return site_released_langs
```

Comment:
```suggestion
        released_languages = self.released_languages

        if settings.FEATURES.get("EDNX_SITE_AWARE_LOCALE", False):
            site_released_langs = getattr(settings, "released_languages", None)            
            released_languages = site_released_langs if site_released_langs else  []
```

and change the line 57 to `languages = [lang.lower().strip() for lang in released_languages.split(',')]`

Thoughts ??

Reply:
good idea, I'll try it

Reply:
I did this:
```
        released_languages = self.released_languages

        if settings.FEATURES.get("EDNX_SITE_AWARE_LOCALE", False):

            site_released_langs = getattr(settings, "released_languages", None)
            released_languages = site_released_langs if site_released_langs else ""

        languages = [lang.lower().strip() for lang in released_languages.split(',')]

        # Put in alphabetical order
        languages.sort()

        return languages
```
I changed `else []` for `else ""` so the `.split()` won't fail

Reply:
For what I tested, the behavior is the same, when `site_released_langs` is `None` this happens in both cases:

![Screenshot from 2020-09-30 11-32-15](https://user-images.githubusercontent.com/64440265/94706840-b2db2b00-0310-11eb-9063-e6143a9eef05.png)
So I guess the change is OK